node_modules/col-assets/lib/api.js

```diff
@@ -144,6 +147,14 @@ var getAsset = module.exports.getAsset = function(ctx, id, opts, callback) {
       'required': false
     });
   }
+  if (opts.includeWhiteboardUsages) {
```

Comment:
When an asset isn't included in an active whiteboard, it could still be part of an exported whiteboard, in which case the user should also not be able to delete it.